_posts/2022-02-09-reviewing-pull-requests.adoc

```diff
@@ -0,0 +1,182 @@
+---
+title: Reviewing pull requests
+layout: post
+author: David Kwon
+description: >-
+  Reviewing pull requests with Eclipse Che.
+categories: []
+keywords: ['GitHub', 'pull request', 'review', 'reviewng', 'PR']
+slug: /@david.kwon/reviewing-pull-requests
+---
+
+Reviewing pull requests for a GitHub project integrated with {prod}.
+
+=== Using {prod} to review pull requests
+
+As long as you're using a supported web browser like Google Chrome, {prod} can make it possible to review a pull request (PR) without setting up runtimes, build tools, or any additional software on the local machine.
+
+This blog post will showcase the typical workflow on how PRs can be reviewed using {prod} for a GitHub project that is well-integrated with {prod}.
+The GitHub project used in this blog post is located here: link:https://github.com/che-incubator/quarkus-api-example[www.github.com/che-incubator/quarkus-api-example].
+
+Please refer to these references on how to set up your project with {prod}:
+
+* <<../../01/11/@ilya.buziuk-contributing-for-the-first-time-to-a-project#set-up-project,How can maintainers set up their projects to use Eclipse Che?>>
+* link:https://www.eclipse.org/che/docs/che-7/end-user-guide/authoring-devfiles-version-2[Authoring a devfile v2]
+
+NOTE: It is crucial that your project contains a well-defined devfile.yaml file to make the most out of your development experience with {prod}. The devfile should define your project's development environment such as development commands, containers, endpoints etc.
+
+In the following scenario, {prod} has been deployed alongside the {devworkspace} engine, which is currently not in use on link:https://workspaces.openshift.com/[Eclipse Che Hosted by Red Hat].
+
+To open the example GitHub project on Eclipse Che Hosted by Red Hat, please use link:https://github.com/che-incubator/quarkus-api-example/tree/devfilev1[https://github.com/che-incubator/quarkus-api-example/tree/devfilev1] for devfile v1 support.
```

Comment:
Matter of preference, but I would probably use WARN: for this paragraph, and stress out that in the article we are using devfile v2, and that devfile v1 is obsolete / deprecated

Reply:
Great idea, my latest push uses WARNING.